docker-hub/vulnerability-scanning.md

```diff
@@ -10,15 +10,34 @@ title: Hub Vulnerability Scanning
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
 %}
 
-Docker Hub Vulnerability Scanning enables you to automatically scan Docker images for vulnerabilities using Snyk. This uses the same technology as the [docker scan](../engine/scan/index.md) command.
+Docker Hub Vulnerability Scanning enables you to automatically scan Docker
+images for vulnerabilities using Snyk. This uses the same technology as the
+[docker scan](../engine/scan/index.md) command.
+
+> When you enable Hub Vulnerability Scanning, you can also see whether your
+> images are affected by Log4Shell (CVE-2021-44228). For more information, see
+> [Scan images](#scan-images).
 
 When you push an image to Docker Hub after enabling vulnerability scanning, Docker Hub automatically scans the image to identify vulnerabilities in your container images. Vulnerability Scanning allows developers and development teams to review the security state of the container images and take actions to fix issues identified during the scan, resulting in more secure deployments. The scan result includes the source of the vulnerability, such as OS packages and libraries, version in which it was introduced, and a recommended fixed version (if available) to remediate the vulnerabilities discovered.
 
 ## Scan images
 
-Hub Vulnerability scanning allows repository owners and administrators of a Docker Pro, Team, or a Business tier to enable and disable scanning.
-
-In addition, repository owners in a Docker Pro subscription and team members in a Team, or a Business subscription can view the detailed scan reports. When scanning is enabled on a specific repository, anyone with push access can trigger a scan by pushing an image to Docker Hub.
+> **Log4j 2 CVE-2021-44228**
+>
+> Versions of `docker scan` earlier than `v0.11.0` are not able to detect [Log4j 2
+> CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228){:
+> target="_blank" rel="noopener" class="_"}. You must update your Docker
```

Comment:
This is confusing as you dont have to update for Docker Hub, we do. The important issue is older scans wont pick it up.

Reply:
OK. Will remove the statement on updating the installation. How about 

"Docker Hub security scans triggered after **1700 UTC 13 December 2021**  are now correctly identifying the Log4j2 CVE. Scans before this date **do not** currently reflect the status of this vulnerability. Therefore, we recommend that you trigger scans by pushing the image to Docker Hub to view the status of Log4j2 CVE in the vulnerability report."

Reply:
Updated